### DISCUSSION TITLE
Budibase releases and Docker versions are confusing

### DISCUSSION THREAD

**Discussion by vorotech** [Budibase/budibase#3757] [General]
Hi here 👋 

I'd like to understand how budibase released versions are linked to Docker image versions.
We are using the docker-compose for on-premise setup.
The images in the docker-compose.yml file specified as `budibase.docker.scarf.sh/budibase/apps` and `budibase.docker.scarf.sh/budibase/worker`.
When I check the version of the app in the UI, it shows current version `1.0.6` and no updates available.
When I check the releases https://github.com/Budibase/budibase/releases/ it shows `v0.9.187` as a latest release (despite the `v1.0.13` exists as a tag)
So what would happen when I next time make the `docker pull` (as part of the docker-compose I mean) ?

Thanks

**Comment by mjashanks** [comment]
I agree that it's hard to see what the latest version is. I have this problem as well - this is something that we need to sort out.

The latest version is `1.0.11` (`1.0.13`). docker pull should bring you to `1.0.11` (it did fo me just now).

But I hear you - this needs to be more clear - thanks for pointing it out.
